project-base/.env.test

```diff
@@ -1 +1,14 @@
 # define your env variables for the test environment here
+
+DATABASE_HOST=postgres
+DATABASE_PORT=5432
+DATABASE_NAME=shopsys-test
+DATABASE_USER=root
+DATABASE_PASSWORD=root
+SELENIUM_SERVER_HOST=selenium-server
+OVERWRITE_DOMAIN_URL=http://webserver:8080
+MAILER_TRANSPORT=smtp
+MAILER_HOST=smtp-server
+MAILER_USER=
+MAILER_PASSWORD=
+SHOPSYS_CONTENT_DIR_NAME=content-test
```

Comment:
I'm missing here variables with default prefixes for Elastic and Redis.

Reply:
I am resolving this one. Pleaso continue discussion above https://github.com/shopsys/shopsys/pull/2055#discussion_r502387455